src/foam/core/lib.js

```diff
@@ -84,7 +84,6 @@ Object.defineProperty(
 foam.assert = function assert(cond) {
   if ( ! cond ) {
     console.assert(false, Array.from(arguments).slice(1).join(' '));
-    console.trace();
```

Comment:
I really like when these show up - does the trace get printed somewhere else?

Reply:
Oh lol, turns out I added that line myself and forgot. I think I remember adding it because there wasn't a trace printed for this anywhere, but is there perhaps a better way to do this?

Reply:
I removed this out of personal preference, but if people prefer to keep it then I can revert this change. Personally I find the stack trace takes up too much space in the console.

Reply:
Addressed by 1d8e5c759.